README.md

```diff
@@ -15,18 +15,37 @@ level of reference detail, i.e. only references for actually used
 functionality will be presented back if software provides multiple
 citeable implementations.
 
-![Example](examples/duecredit_example.gif)
+## Installation
+
+Duecredit is easy to install via pip, simply type:
+ 
+ `pip install duecredit`
+
+## Examples
 
-duecredit 101
-=============
+### To cite the modules you are using 
 
 You can already start "registering" citations using duecredit in your
 Python modules and even registering citations (we call this approach "injections")
 for modules which do not (yet) use duecredit.  duecredit will remain an optional
 dependency, i.e. your software will work correctly even without duecredit installed.
 
-"Native" use of duecredit (recommended)
----------------------------------------
+For example, list of references to the modules `yourproject` uses ca be produced with few simple commands:
```

Comment:
Sounds good. Feel free to make changes! Let me know if I have commit again (I don't think so, right?)